docsrcs/yui-ibc-solidity/docs/minitoken/contract.md

```diff
@@ -0,0 +1,318 @@
+---
+sidebar_position: 3
+---
+
+# Create a contract
+
+We will implement a token that can be transferred between two ledgers using IBC.
+
+There is a token standard called [ICS-20](https://github.com/cosmos/ibc/tree/master/spec/app/ics-020-fungible-token-transfer),
```

Comment:
ICS-20 might be better described the token "transfer" standard, not just "token".
Therefore, we should modify the sentence "but we will use a simple one here" to "but we don't support the standard here"?